ui/components/ui/detailed-error-message/detailed-error-message.component.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DetailedErrorMessage = (props) => {
+  const { errorMessage, onErrorMessageClick, linkText } = props;
+
+  return (
+    <div className="error-message">
+      <img
+        src="./images/alert-red.svg"
+        alt=""
+        className="error-message__icon"
+      />
+      <div className="error-message__text">
+        {errorMessage}
+        <a className="error-message__link" onClick={onErrorMessageClick}>
```

Comment:
We should make this a `<Button type="link">` component for the sake of accessibility.